src/main/api/catalog.ts

```diff
@@ -85,17 +85,35 @@ export class CatalogApi implements ICatalogApi {
             }
         }
 
+        const isAudiobook = (item: PublicationView) =>
+            item.RDFType && /http[s]?:\/\/schema\.org\/Audiobook$/.test(item.RDFType);
+
+        const lastAdded = lastAddedPublicationViews.filter((item) => !isAudiobook(item));
+        const lastRead = lastReadPublicationViews.filter((item) => !isAudiobook(item));
+        const audiobooks = lastAddedPublicationViews.filter(isAudiobook);
+        const lastAudiobooks = lastReadPublicationViews.filter(isAudiobook);
+
         // Dynamic entries
         let entries: CatalogEntryView[] = [
             {
                 title: __("catalog.entry.continueReading"),
-                totalCount: lastReadPublicationViews.length,
-                publicationViews: lastReadPublicationViews,
+                totalCount: lastRead.length,
+                publicationViews: lastRead,
             },
             {
                 title: __("catalog.entry.lastAdditions"),
-                totalCount: lastAddedPublicationViews.length,
-                publicationViews: lastAddedPublicationViews,
+                totalCount: lastAdded.length,
+                publicationViews: lastAdded,
+            },
+            {
+                title: __("catalog.entry.continueAudiobooks"),
```

Comment:
For consistency with the existing keys, I would name this `continueReadingAudioBooks` (to mirror `catalog.entry.continueReading`)